pkg/storkctl/clusterpair.go

```diff
@@ -267,10 +274,14 @@ func newCreateClusterPairCommand(cmdFactory Factory, ioStreams genericclioptions
 				return
 			}
 
-			if len(sIP) > 0 || len(sPort) > 0 || len(dIP) > 0 || len(dPort) > 0 {
+			if len(sIP) > 0 || len(sPort) > 0 || len(dIP) > 0 || len(dPort) > 0 || len(srcToken) > 0 || len(destToken) > 0 {
```

Comment:
Lets keep the option for the user to specify `srcToken` & `destToken` if for some reason the automatic token fetch fails. You can still mark it as deprecated in the CLI

Reply:
srcToken and destToken is deprecated now (it will show as deprecated but values will be honoured but in --help these options will not show up.

Reply:
As discussed , removing `MarkDeprecated` for `srcToken` and `destToken`. These will be optional params.